docs/source/commands.rst

```diff
@@ -33,3 +35,39 @@ Command-line options
 .. option:: --app-label <app_label>
 
    Create initial data with the *app_label* provided.
+
+.. _command-generate-pep-pages:
+
+generate_pep_pages
+------------------
+
+This Django management command generates `pages.Page` objects from the output
+of the existing PEP repository generation process. You run it like::
+
+    $ ./manage.py generate_pep_pages
+
+To get verbose output, specify :option:`--verbosity` option::
+
+    $ ./manage.py generate_pep_pages --verbosity=2
+
+It uses the conversion code in the `peps.converters` module in an attempt to
```

Comment:
Style nit: \`peps.converters\` -> \`\`peps.converters\`\`

Or we can replace it with

```rst
[...] in the ``peps/converters.py`` file [...]
```